docusaurus/docs/create-a-plugin/extend-a-plugin/fetch-data-from-frontend.md

```diff
@@ -169,23 +187,55 @@ export function ConfigEditor(props: Props) {
         width={40}
       />
     </InlineField>
+    {/* The rest of your configuration page form */}
   );
 }
 ```
 
-Once the field is set correctly, you may use it inside your data source. Following the previous example for the data source code, you can now simply use it like so:
+### Step 3: Fetch data from your frontend code
+
+In your data source plugin, you can now fetch data by using the proxy URL.
+
+See the [previous example](#step-2-create-your-configuration-page) as the code is the same.
+
+## Use the data proxy within an app plugin
+
+The setup of routes in your `plugin.json` metadata remains the same as in a data source plugin; however, since app plugins don't receive the URL as part of the props, the URL is constructed like this:
```

Comment:
```suggestion
The setup of routes in your `plugin.json` metadata remains the same as in a data source plugin. However, since app plugins don't receive the URL as part of the props, the URL is constructed like this:
```